owl2nl/src/main/java/org/aksw/owl2nl/OWLAxiomConverter.java

```diff
@@ -341,18 +343,29 @@ public void visit(OWLAnnotationPropertyRangeAxiom axiom) {
 	@Override
 	public void visit(OWLDeclarationAxiom axiom) {
 	}
-	
+
 	public static void main(String[] args) throws Exception {
-		ToStringRenderer.getInstance().setRenderer(new DLSyntaxObjectRenderer());
-		String ontologyURL = "http://www.cs.man.ac.uk/~stevensr/ontology/family.rdf.owl";// subproperties of the form 'isSomething'
+
+	}
+
+	public String GetOntology(String path) throws Exception {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.